### PR TITLE
Feature/portoaj/stage 6 component

### LINKED ISSUE #53: Foster Profile Component (Stage 6)

### PR DESCRIPTION
**Pull request by portoaj**
### Administrative Info
Issue 53

### Changes
- Added internalNotes field to User model
- Added FosterProfile component
- Added FosterProfile component to step 6 of application flow

### Note
- Still need to make step 6 get actual user for both Foster Profile and Dog Profile

### Testing
How did you confirm your changes worked? 
- Made an application at step 6 and looked at result

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

image

### CODE REVIEW

File: client/src/components/FosterProfile.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  width: min-content;
+`;
+const Title = styled.div`
+  color: white;
+  font-size: 33px;
+  padding-bottom: 20px;
+`;
+const CardTitle = styled.div`
+  font-size: 30px;
+`;
```
Comment (wllmwu) [inline client/src/components/FosterProfile.js:33]: Add a bit of top padding, maybe 10px or so

File: client/src/components/FosterProfile.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+const NotesTitle = styled.span`
+  font-size: 16px;
+`;
```
Comment (wllmwu) [inline client/src/components/FosterProfile.js:36]: Add a bit of bottom padding, maybe also 10px

File: client/src/components/FosterProfile.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  border-radius: 35px;
+  width: min-content;
+  text-align: center;
+  padding: 5px 60px 60px 60px;
+`;
+const ContentBackground = styled.div`
+  background-color: white;
+  border-radius: 13.85px;
+  width: min-content;
+`;
```
Comment (wllmwu) [inline client/src/components/FosterProfile.js]: Change border radius to 15px

File: client/src/components/FosterProfile.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  font-size: 30px;
+`;
+const NotesTitle = styled.span`
+  font-size: 16px;
+`;
+const ContentBox = styled.div`
+  width: min-content;
+  border: 3px solid #8dc442;
+  border-radius: 23.6px;
+`;
```
Comment (wllmwu) [inline client/src/components/FosterProfile.js:41]: Make this border radius match that of `ContentBackground` (I suggested 15px)

File: client/src/components/FosterProfile.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+      </ContentBackground>
+    </Background>
+  );
+}
+/*
+function FosterProfile(props) {
+  return (
+    <Background>
+      <Title>Foster Profile</Title>
+      <ContentBackground>
```
Comment (wllmwu) [inline client/src/components/FosterProfile.js:86]: You can delete this unused part or add a comment saying it's the original design

File: client/src/pages/Application/AdminView/Step6.js [hunk 6]
```diff
@@ -31,9 +32,12 @@ const Column = styled.div`
...
     <Column>
-      <Meetings
-        title="Foster  Profile"
-        // TODO
+      <FosterProfile
+        name="Amy C."
+        ambassadorName="Kristin"
+        coordinatorName="Andy L."
+        fosterHistory={[{ name: "Lolita" }, { name: "Flower" }]}
+        internalNotes="Lorem ipsum internal notes here"
```
Comment (wllmwu) [inline client/src/pages/Application/AdminView/Step6.js]: Pass in data from the backend once you have Parth's stuff merged into your branch

**Review by wllmwu — COMMENTED** [review]
Just reviewed `FosterProfile` for now since I think most of the other changed files are from Parth's branch. Once Parth's PR is merged, go ahead and merge `main` into your branch and I'll approve this 👍

**Comment by portoaj** [comment]
What it looks like right now:
image

Not sure what to write for status field

File: client/src/pages/Application/AdminView/Step6.js [hunk 7]
```diff
@@ -50,12 +88,12 @@ function FosterAndDogInformation() {
           <Column>
             <StatusUpdate
               title="Ambassador Contact Info"
-              ambassador="Dhanush"
-              phone="123-456-7890"
-              email="test@tap.com"
+              ambassador={ambassadorName}
+              phone={ambassadorPhone}
+              email={ambassadorEmail}
               status="Not Matched"
```
Comment (wllmwu) [inline client/src/pages/Application/AdminView/Step6.js]: Just remove this status field in the admin thingy

**Review by wllmwu — APPROVED** [review]
Looks good, just remove the admin status thing and then go ahead and merge 👌

**Pull request merged**